ui/admin/app/components/form/field/list-wrapper/key-value/index.hbs

```diff
@@ -10,15 +10,14 @@
         <H.Th id={{keyHeaderID}}>{{t 'form.key.label'}}</H.Th>
         <H.Th id={{valueHeaderID}}>{{t 'form.value.label'}}</H.Th>
         <H.Th>
-          {{#if @removeOptionByIndex}}
+          {{#if this.removeOptionByIndex}}
```

Comment:
I don't think you need these if statements anymore since you'll always have it now